webpack/scenes/BootedContainerImages/BootedContainerImagesPage.js

```diff
@@ -0,0 +1,242 @@
+    defaultParams,
+    apiOptions,
+    setAPIOptions: response.setAPIOptions,
+  });
+  const onSort = (_event, index, direction) => {
+    setParamsAndAPI({
+      ...params,
+      order: `${Object.keys(columns)[index]} ${direction}`,
+    });
+  };
+  const { pfSortParams } = useTableSort({
+    allColumns: Object.keys(columns).map(k => columns[k].title),
+    columnsToSortParams,
+    onSort,
+  });
+  const expandedImages = useSet([]);
+  const imageIsExpanded = bootcBootedImage => expandedImages.has(bootcBootedImage);
+  const STATUS = {
+    PENDING: 'PENDING',
+    RESOLVED: 'RESOLVED',
+    ERROR: 'ERROR',
+  };
+
+  const {
+    response: {
+      results = [],
+      subtotal,
+      message: errorMessage,
+    },
+    status = STATUS.PENDING,
+  } = response;
+
+  const [columnNamesKeys, keysToColumnNames] = getColumnHelpers(columns);
+  const onPagination = (newPagination) => {
+    setParamsAndAPI({ ...params, ...newPagination });
+  };
+  const bottomPagination = (
+    <Pagination
+      key="table-bottom-pagination"
+      page={params.page}
+      perPage={params.perPage}
+      itemCount={subtotal}
+      onChange={onPagination}
+      updateParamsByUrl
+    />
+  );
+
+  const getColumnWidth = (key) => {
+    if (key === 'bootc_booted_image') return 40;
+    if (key === 'digest') return 15;
+    return 45;
+  };
+
+  return (
+    <TableIndexPage
+      apiUrl={BOOTED_CONTAINER_IMAGES_API_PATH}
+      apiOptions={apiOptions}
+      header={__('Booted container images')}
+      createable={false}
+      isDeleteable={false}
+      controller="/katello/api/v2/host_bootc_images"
+    >
+      <>
+        <TableComposable variant="compact" ouiaId="booted-containers-table" isStriped>
+          <Thead>
+            <Tr ouiaId="table-header">
+              <>
+                <Th />
+                {columnNamesKeys.map(k => (
+                  <Th
+                    width={getColumnWidth(k)}
+                    key={k}
+                    sort={
+                      Object.values(columnsToSortParams).includes(k) &&
+                      pfSortParams(keysToColumnNames[k])
+                    }
+                  >
+                    {keysToColumnNames[k]}
+                  </Th>
+                ))}
+              </>
+            </Tr>
+          </Thead>
+          <Tbody>
+            {status === STATUS.PENDING && results.length === 0 && (
+              <Tr ouiaId="table-loading">
+                <Td colSpan={100}>
+                  <EmptyPage
+                    message={{
+                      type: 'loading',
+                      text: __('Loading...'),
+                    }}
+                  />
+                </Td>
+              </Tr>
+            )}
+            {!status === STATUS.PENDING &&
+              results.length === 0 &&
+              !errorMessage && (
+                <Tr ouiaId="table-empty">
+                  <Td colSpan={100}>
+                    <EmptyPage
```

Comment:
I'm not sure why, but this empty state page does not want to get rendered. The loading page renders even, just not this one.

Reply:
see above, maybe that's it?